pkg/kopia/stream.go

```diff
@@ -36,18 +36,28 @@ const (
 	pullRepoPurpose = "kando location pull"
 )
 
+// SnapshotInfo tracks kopia snapshot information produced by a kando command in a phase
+type SnapshotInfo struct {
+	// BackupID is the snapshot ID produced by kopia snapshot operation
+	BackupID string `json:"backupID"`
+	// LogicalSize is the sum of cached and hashed file size in bytes
+	LogicalSize string `json:"logicalSize"`
```

Comment:
[nit] I prefer using an int in our internal structures and letting serialization handle parsing